### ISSUE TITLE
Normalize Softmax Inputs

### ISSUE THREAD

**Issue by LittleBigPhil** [LittleBigPhil/ConstructionProblemRL#16]
Use a momentum approximator to determine the average output from the neural network, and use this as an offset for the softmax inputs. This should be set on the reset of the softqueue.
